apis/tempo/v1alpha1/tempostack_types.go

```diff
@@ -133,6 +133,13 @@ type ObservabilitySpec struct {
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Tracing Config"
 	Tracing TracingConfigSpec `json:"tracing,omitempty"`
+
+	// Datasources defines the datasources configuration for operands.
+	//
+	// +optional
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Datasources Config"
+	Datasources DatasourcesConfigSpec `json:"datasources,omitempty"`
```

Comment:
I'm wondering if we should rename this to `grafana` instead. Then we'll have `metrics`, `tracing` and `grafana` in `.spec.observability`. Wdyt @pavolloffay?

Reply:
+1

Reply:
Renamed it.